help/destinations/catalog/personalization/algolia-user-segments.md

```diff
@@ -0,0 +1,147 @@
+| [!DNL Segmentation Service] | ✓ | Audiences generated through the Experience Platform [Segmentation Service](../../../segmentation/home.md). |
+
+{style="table-layout:auto"}
+
+## Export type and frequency {#export-type-frequency}
+
+Refer to the table below for information about the destination export type and frequency.
+
+| Item | Type | Notes |
+|---------|----------|---------|
+| Export type | **[!DNL Profile request]** | You are requesting all the audiences that are mapped in the Algolia User Segments destination for a single profile.|
+| Export frequency | **[!UICONTROL Streaming]** | Streaming destinations are "always on" API-based connections. As soon as a profile is updated in Experience Platform based on audience evaluation, the connector sends the update downstream to the destination platform. Read more about [streaming destinations](/help/destinations/destination-types.md#streaming-destinations).|
+
+{style="table-layout:auto"}
+
+## Connect to the destination {#connect}
+
+>[!IMPORTANT]
+>
+>To connect to the destination, you need the **[!UICONTROL View Destinations]** and **[!UICONTROL Manage and Activate Dataset Destinations]** [access control permissions](/help/access-control/home.md#permissions). Read the [access control overview](/help/access-control/ui/overview.md) or contact your product administrator to obtain the required permissions.
+
+To connect to this destination, follow the steps described in the [destination configuration tutorial](../../ui/connect-destination.md). In the destination configuration workflow, fill in the fields listed in the two sections below.
+
+### Authenticate to destination {#authenticate}
+
+To authenticate to the destination, fill in the required fields and select **[!UICONTROL Connect to destination]**.
+
+#### New account {#new-account}
+
+| Field | Description |
+| --- | --- |
+| [!UICONTROL Application ID] | The [!DNL Algolia] application ID is a unique identifier assigned to your [!DNL Algolia] account. |
+| [!UICONTROL API Key] | The [!DNL Algolia] API Key is a credential used to authenticate and authorize API requests to [!DNL Algolia]'s search and indexing services. |
+
+For more information on these credentials, see the [!DNL Algolia] [authentication documentation](https://www.algolia.com/doc/tools/cli/get-started/authentication/).
+
+![New Account](../../assets/catalog/personalization/algolia-user-segments/connection.png)
+
+After connection to destination is successful, enter the below information to complete the account creation.
+
+| Field | Description |
+| --- | --- |
+| [!UICONTROL Name] | Fill in the preferred name for this destination. |
+| [!UICONTROL Description] | Short explanation of the destination's purpose. |
+| [!UICONTROL Region] | The options are **US** or **EU**. Select the region where the customer data is stored. |
+
+![Account details](../../assets/catalog/personalization/algolia-user-segments/account.png)
+
+### Existing account {#existing-account}
```

Comment:
The purpose of this page is to guide users through creating a new account. There is no need for this section here. This is part of the activation workflow for existing accounts, which is already documented in other sections of our docs.